nixos/doc/manual/release-notes/rl-1609.xml

```diff
@@ -202,7 +202,7 @@ following incompatible changes:</para>
 
   <listitem>
     <para>The directory container setuid wrapper programs,
-    <filename>/var/setuid-wrappers</filename>, <link
+    <filename>/var/permissions-wrappers</filename>, <link
```

Comment:
I don't think we'll backport this to 16.09 so I don't think we should update the release notes here.